lib/cocoapods/sandbox.rb

```diff
@@ -352,20 +350,24 @@ def remove_checkout_source(name)
     #         The name of the Pod.
     #
     # @param  [#to_s] path
-    #         The local path where the Pod is stored.
+    #         The path to the local Podspec
     #
     # @param  [Bool] was_absolute
     #         True if the specified local path was absolute.
     #
+    # @param  [AbstractExternalSource] source
+    #         The source which provided the podspec
+    #
```

Comment:
You may have removed this param at a later date

Reply:
Indeed, good catch!